internal/app/wwclient/root.go

```diff
@@ -16,8 +16,8 @@ import (
 
 	"github.com/coreos/go-systemd/daemon"
 	"github.com/google/uuid"
-	"github.com/hpcng/warewulf/internal/pkg/pidfile"
 	warewulfconf "github.com/hpcng/warewulf/internal/pkg/config"
```

Comment:
It looks to me like the change from #781 _is_ present here, so I'm confused by the PR description.